tools/sidekick/config.json

```diff
@@ -10,7 +10,18 @@
       "url": "/aem-asset-selector",
       "isPalette": true,
       "includePaths": [ "**.docx**" ],
-      "paletteRect": "top: auto; bottom: 20px; left: 20px; width:1000px; height:700px"
+      "paletteRect": "top: 50px; bottom: 10px; right: 10px; left: auto; width:400px; height: calc(100vh - 60px)"
+    },
+    {
+      "id": "asset-library-stage",
```

Comment:
Do we need `asset-library-stage` ?

Reply:
It's not required. This was more just for convenience for us, since we've been using a stage environment so that we can verify everything end-to-end, including using one of asset link's client IDs. You can see the document that corresponds with this button [here](https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7BAFAFC263-CAED-4644-B5E8-94D635551841%7D&file=aem-asset-selector-stage.docx&action=default&mobileredirect=true).

FWIW, the stage button will only show up for anyone working on the asset selector code itself - it wouldn't need to be there for customers. But I can remove if and we can add the button locally if we don't want to include it here.

Reply:
Yes probably its fine to have if it helps with testing. Customer are going to add their own sidekick configs and we'd document only the prod one.